src/api/app/datatables/maintenance_incident_datatable.rb

```diff
@@ -26,8 +26,7 @@ def data
         category: category_cell(record, patchinfo),
```

Comment:
Do it make sense to have in category, something that isn't a category?

Reply:
So far it made sense for the maintenance team (bento UI). I am reluctant to change this without talking with them first.

Note that this is already part of the view, old and new UI.

Reply:
ah ok, sorry for the noise